albo.py

```diff
@@ -24,44 +13,41 @@ def send_telegram_message(text):
     URL = "https://api.telegram.org/bot" + TOKEN + "/sendMessage?&parse_mode=markdown&" + urlencode(params, quote_via=quote_plus)
     r = requests.get(url = URL)
 
+def get_headers(table):
```

Comment:
the type is missing
```suggestion
def get_headers(table: TYPE) -> list:
```

I don't know the TYPE but if you know add it please

for the return type you could use `-> List[str]` it is better

Reply:
I'm more keen on removing the get_headers method completely and just use list comprehension